lms/templates/shoppingcart/shopping_cart.html

```diff
@@ -9,140 +9,143 @@
 
 %>
 
+<section class="wrapper confirm-enrollment shopping-cart cart-view">
+  <%block name="custom_content">
+
+  <div class="container">
+    % if shoppingcart_items:
+          <%block name="billing_details_highlight">
+           % if order.order_type == 'business':
+              <li class="billing">${_('Billing Details')}</li>
+           % else:
+              <li class="billing hidden">${_('Billing Details')}</li>
+           % endif
+          </%block>
+
+       % if is_course_enrollment_closed:
+        <p id="expiry-msg">${_('{course_names} has been removed because the enrollment period has closed.').format(course_names=appended_expired_course_names)}</p>
+       % endif:
+   <%
+      discount_applied = False
+      order_type = 'personal'
+   %>
+
+    <article class="wrapper confirm-enrollment shopping-cart cart-view" aria-label="${_('Shopping Cart')}">
+      % for item, course in shoppingcart_items:
+         % if loop.index > 0 :
+          <hr>
+        %endif
+        % if item.order.order_type == 'business':
+         <% order_type = 'business' %>
+        %endif
+       <div class="user-data">
+        <div class="clearfix">
+        <div class="image">
+
+             <img style="width: 100%; height: 100%;" src="${course_image_url(course)}"
```

Comment:
I know this is not your wrongdoing, but shouldn't these go to style, and alt be internationalized.